.github/actions/prepare-discovery/action.yml

```diff
@@ -0,0 +1,15 @@
+# A composite action to install discovery
+
+name: Prepare discovery
+
+description: Install discovery
+
+runs:
+  using: composite
+  steps:
+    - name: Install discovery
+      run: |
+        wget https://github.com/mitchnegus/minimega-discovery/releases/download/firewheel-debian_faed761/discovery.deb
```

Comment:
I created a deb file for discovery based on my best attempts to update the [discovery](https://github.com/sandia-minimega/discovery/tree/master) source code for a modern version of Go. Pending the results of sandia-minimega/discovery#15, we should be able to remove the dependence on my personal fork.